src/caustics/lenses/pseudo_jaffe.py

```diff
@@ -149,21 +142,7 @@ def mass_enclosed_2d(
         surface_density_0 = self.get_convergence_0(
             z_s, params
         ) * self.cosmology.critical_surface_density(z_l, z_s, params)
-        return (
-            2
-            * pi
-            * surface_density_0
-            * core_radius
-            * scale_radius
-            * (d_l * arcsec_to_rad) ** 2
-            / (scale_radius - core_radius)
-            * (
-                (core_radius**2 + theta**2).sqrt()
-                - core_radius
-                - (scale_radius**2 + theta**2).sqrt()
-                + scale_radius
-            )
-        )
+        return (2 * pi * surface_density_0 * core_radius * scale_radius * (d_l * arcsec_to_rad) ** 2 * ((core_radius**2 + theta**2).sqrt() - core_radius - (scale_radius**2 + theta**2).sqrt() + scale_radius) / (scale_radius - core_radius))  # fmt: skip
```

Comment:
I think this equation is a bit too long and is confusing me. 

Can we reformat it in a way that clarifies what is being computed and the units?

Below, I am attempting to do this although I am note sure this is right. 

```
        theta = theta + self.s # arcsec
        D_A = self.cosmology.angular_diameter_distance(z_l, params)  # Mpc
        Sigma_crit = self.cosmology.critical_surface_density(z_l, z_s, params)  # M_sun / Mpc^2
        Sigma_0 = self.get_convergence_0(z_s, params) * Sigma_crit  # M_sun / Mpc^ 2
        xi = D_A * theta * arcsec_to_rad # distance perpendicular to the line-of-sight (Mpc)
        mass = 2 * pi * Sigma_0 * D_A**2 * (core_radius * scale_radius * arcsec_to_rad)**2
        ratio = (core_radius**2 + theta**2).sqrt() - core_radius - (scale_radius**2 + theta**2).sqrt() + scale_radius) / (scale_radius - core_radius) #  no units
# The rest of the equation in terms of variables above
```